client/docs/swagger-ui/swagger.yaml

```diff
@@ -29728,6 +29740,9 @@ definitions:
                 created_at:
                   type: string
                   format: int64
+                closed_on:
+                  type: string
+                  format: int64
```

Comment:
Is this set in `x/escrow`?

Reply:
x/market since it is part of lease.

Reply:
It just looks like it's part of the escrow object:

![2021-09-29-110333_632x358_scrot](https://user-images.githubusercontent.com/12553/135324187-ae6a2415-e92c-43d3-8bad-4dc4e9f92c57.png)

This was the confusing bit to me.

Reply:
OK....that is weird. I have no idea. It's on the lease message itself. Let me look at what is getting generated for the swagger docs.